modules/ballerina-http/src/main/java/org/ballerinalang/net/http/Constants.java

```diff
@@ -184,21 +183,29 @@
     public static final int CIPHERS_INDEX = 5;
     public static final int SSL_PROTOCOL_INDEX = 6;
 
-    //Request struct indexes
-    public static final int REQUEST_HOST_INDEX = 0;
-    public static final int REQUEST_PATH_INDEX = 1;
-    public static final int REQUEST_METHOD_INDEX = 2;
-    public static final int REQUEST_VERSION_INDEX = 3;
-    public static final int REQUEST_USER_AGENT_INDEX = 4;
-    public static final int REQUEST_REST_URI_POSTFIX_INDEX = 5;
-    public static final int REQUEST_PORT_INDEX = 0;
-    public static final int REQUEST_HEADERS_INDEX = 0;
-
-    //Response struct indexes
-    public static final int RESPONSE_STATUS_CODE_INDEX = 0;
-    public static final int RESPONSE_REASON_PHRASE_INDEX = 0;
-    public static final int RESPONSE_SERVER_INDEX = 1;
-    public static final int RESPONSE_HEADERS_INDEX = 0;
+    public static final int HTTP_MESSAGE_INDEX = 0;
+    public static final int ENTITY_INDEX = 1;
+    public static final int ENTITY_BODY_REQUIRED_INDEX = 0;
```

Comment:
Do we need the unused variable ENTITY_BODY_REQUIRED_INDEX?

Reply:
Removed